css/waiting.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans|Roboto|Droid+Serif');
```

Comment:
Nitpicks: whitespace and formatting in this file is a little inconsistent. Maybe run it through a CSS formatter?

Reply:
yeah, it's coming from designer.  i'll run it through a formatter

Reply:
fixed in commit: 4f8224a0174f523fece7fb8c9fad6bdc3d7fd920

Reply:
👍 it is beautiful

Reply:
I would prefer to avoid using external assets if possible, especially for something that requires fastest load time possible.

Reply:
@hshaosf d1f86296c79dd3ad85a3876c8c43ed40af6efba3